website/src/pages/index.js

```diff
@@ -35,7 +35,7 @@ function HomepageHeader() {
         <div className={styles.buttons}>
           <Link
```

Comment:
This should be an <a> tag. Link is a react component, it may not be consistent with standard HTML tag behavior.